inference-engine/thirdparty/clDNN/src/network.cpp

```diff
@@ -418,6 +418,8 @@ void network::set_output_memory(const primitive_id& id, memory::ptr mem_new) {
 
     for (auto& prim : o_iter->second) {
         prim->set_output_memory(eng.reinterpret_buffer(*mem_new, prim->output_memory().get_layout()), false);
+        if (!_reset_arguments)
```

Comment:
Can I ask why this is done when _reset_arguments is false? 
Shouldnt it be when _reset_arguments is true?